### PR TITLE
feat(e2e-tests): added playwright e2e workflow

### PR DESCRIPTION
**Pull request by jordan-badger**

### CODE REVIEW

File: .github/workflows/e2e-tests.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+name: ecmascript-unit-test
```
Comment (kieranbrown) [inline .github/workflows/e2e-tests.yaml:1]: let's keep a consistent name throughout, think this is just a typo anyway?

File: .github/workflows/e2e-tests.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  e2e-tests:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
```
Comment (kieranbrown) [inline .github/workflows/e2e-tests.yaml:15]: this is outdated, `v3` is the latest

File: .github/workflows/e2e-tests.yaml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+      - uses: actions/setup-node@v1
```
Comment (kieranbrown) [inline .github/workflows/e2e-tests.yaml:17]: also outdated, `v3` is the latest

**Review by kieranbrown — CHANGES_REQUESTED** [review]

File: .github/workflows/e2e-tests.yaml (hunk 1 above)
Comment (jordan-badger) [reply .github/workflows/e2e-tests.yaml]: It was a typo. I changed it everywhere but here

File: .github/workflows/e2e-tests.yaml (hunk 2 above)
Comment (jordan-badger) [reply .github/workflows/e2e-tests.yaml]: Sorted

File: .github/workflows/e2e-tests.yaml (hunk 3 above)
Comment (jordan-badger) [reply .github/workflows/e2e-tests.yaml:17]: sorted

**Pull request merged**